src/cores/events/EventsList.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { addEvent, resetEvents, setAmount } from '../../features/events/eventSlice'
+import * as eventsServices from '../../services/events.services'
+import Event from './Event'
+import Pagination from './Pagination'
+
+function EventsList() {
+    const postPerPage = 9;
+    const page = useSelector(state => state.events.page);
+    const events = useSelector(state => state.events.events);
+    const amount = useSelector(state => state.events.amount)
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(resetEvents());
+        const getEvents = async () => {
+            try {
+                const data = await eventsServices.fetchEvents(page);
+                data.events.forEach(event => {
+                    dispatch(addEvent(event));
+                });
+                dispatch(setAmount(data.paginate.total));
+            } catch(e) {
+                console.log(e);
+            }
+        }
+        getEvents();
+    }, [dispatch, page, amount])
```

Comment:
Vale la pena hacer una funcion siempre que no la uses indiatamente despues de declararla xD, es decir, ¿para que se hizo esto una funcion en primer lugar? (quiza por el await? XDD), mejor invoca todo directamente

```suggestion
    useEffect(async () => {
        dispatch(resetEvents());
        try {
            const data = await eventsServices.fetchEvents(page);
            data.events.forEach(event => {
                dispatch(addEvent(event));
            });
            dispatch(setAmount(data.paginate.total));
        } catch(e) {
            console.log(e);
        }
    }, [dispatch, page, amount])
```